### DISCUSSION TITLE
Tag problem when installing docker compose

### DISCUSSION THREAD

**Discussion by wahve3** [plausible/analytics#4041] [Self-Hosted Support]
![starting-problems-with-docker-compose-v0-1g805rb77gwc1](https://github.com/plausible/analytics/assets/24662784/4961a536-7e8a-4e99-85bf-b6b0d1e441d6)

This problem here with the tags seems to crash the docker compose installation every time I try. I didn't make any other changes to the setup.

**Comment by ruslandoga** [comment]
👋 @wahve3 

It shouldn't crash the app since we handle it in https://github.com/plausible/analytics/blob/805b5acbc78dafef31f52ee607b6faf310811b5a/config/runtime.exs#L170-L189

The cause for this invalid encoding is https://github.com/docker/build-push-action/issues/726

**Reply by ruslandoga** [reply]
Would you be able to provide more the crash logs or docker compose output? Maybe the cause is in something else. Or maybe we don't handle the problem well enough.
